beacon-chain/rpc/debugv1/debug.go

```diff
@@ -30,6 +30,10 @@ func (ds *Server) GetBeaconState(ctx context.Context, req *ethpb.StateRequest) (
 	}, nil
 }
 
+func (ds *Server) GetBeaconStateSsz(ctx context.Context, req *ethpb.StateRequest) (*ethpb.BeaconStateSszResponse, error) {
+	return nil, status.Error(codes.Unimplemented, "Unimplemented")
+}
```

Comment:
This is new?

Reply:
Yes otherwise it fails to compile. I believe the commit I took from ethereumapis (master) has not been reflected in Prysm yet